src/js/smooch.jsx

```diff
@@ -146,11 +147,17 @@ export class Smooch extends Observable {
         }).then((loginResponse) => {
             store.dispatch(setUser(loginResponse.appUser));
             store.dispatch(updateReadTimestamp(getReadTimestamp()));
-            
+
             if (loginResponse.publicKeys) {
                 store.dispatch(setPublicKeys(loginResponse.publicKeys));
-            }
 
+                if (loginResponse.publicKeys.stripe) {
```

Comment:
We should probably find a better way to indicate that stripe is configured for the app in the `init` response